src/Bundle/Form/DataTransformer/ResourceToIdentifierCacheableTransformer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Form\DataTransformer;
+
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Webmozart\Assert\Assert;
+
+final class ResourceToIdentifierCacheableTransformer implements DataTransformerInterface
```

Comment:
```suggestion
final class CachedResourceToIdentifierTransformer implements DataTransformerInterface
```
Though that's just a matter of taste

Reply:
Suggested name is in line with the rest of the sylius, I vote for it as well